plugins/GSdx/GSWndWGL.cpp

```diff
@@ -88,13 +88,20 @@ bool GSWndWGL::CreateContext(int major, int minor)
 		PFNWGLCREATECONTEXTATTRIBSARBPROC wglCreateContextAttribsARB = (PFNWGLCREATECONTEXTATTRIBSARBPROC)wglGetProcAddress("wglCreateContextAttribsARB");
 		if (!wglCreateContextAttribsARB) {
 			fprintf(stderr, "Failed to init wglCreateContextAttribsARB function pointer\n");
-			return false;
+			throw GSDXRecoverableError();
 		}
 
 		HGLRC context30 = wglCreateContextAttribsARB(m_NativeDisplay, NULL, context_attribs);
 		if (!context30) {
-			fprintf(stderr, "Failed to create a 3.x context\n");
-			return false;
+			fprintf(stderr, "Failed to create a 3.x context with standard flags\n");
+			// retry with more compatible option for (Mesa on Windows, OpenGL on WINE)
+			context_attribs[2*2+1] = 0;
```

Comment:
yes because there aren't the same. 5 is 6th postision. 2*2 + 1 is the 3rd row, 2nd column.